frontend/awx/main/ansibleDocsUrls.ts

```diff
@@ -15,4 +15,6 @@ export const ansibleDocUrls: { [key: string]: string } = {
     'https://docs.ansible.com/ansible/latest/collections/community/vmware/vmware_vm_inventory_inventory.html',
   constructed:
     'https://docs.ansible.com/ansible/latest/collections/ansible/builtin/constructed_inventory.html',
+  terraform:
+    'https://github.com/ansible-collections/cloud.terraform/blob/main/plugins/inventory/terraform_state.py',
```

Comment:
I'm not 100% sure we want to link out to a github repo as official documentation.  I'm seeking input from people who would know.

Reply:
Ok, did some digging.  Docs for this are pending publication.  The issue can be tracked here https://issues.redhat.com/browse/ACA-1187.  Might be worth while reaching out on that issue to see if they know the url that would link to terraform docs yet.  Sometimes they do know ahead of time where they will go.  For what its worth I suggested to Lynn Maynard that terraform docs should follow suit with the list of links to docs [here](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/automation_controller_user_guide/index#ref-controller-inventory-sources)

Reply:
@AlexSCorey that issue covers docs for the Terraform State inventory source in AAP, whereas here I think we want to link to the docs for the inventory plugin itself (which is also linked from those docs) so folks can understand what the plugin options are. We haven't done a cloud.terraform release yet with the new plugin so we can't link to published docs...we can either wait until that release happens and link to the plugin docs on Automation Hub, or we can link to the repo.

Reply:
@AlexSCorey I updated the docs link and rebased onto current main to fix a merge conflict in the cypress test file (part of the credential test had been changed since I last worked on this). It looks like there are two failing tests but I don't _think_ they are related to this PR. Let me know if any other changes are needed, thanks!